docs/template.md

```diff
@@ -1,7 +1,25 @@
-# qgrid 
+# qgrid template
 ## markup
 ```html
-<q-grid rows='data'></q-grid>
+<q-grid rows="data" >
+        <q-grid:head> 
+            <q-grid:td> 
+                <i title="{{$column.name}}">{{$column.label}}</i>
+            </q-grid:td>
+        </q-grid:head>
+        <q-grid:body>
+            <q-grid:tr>
+            </q-grid:tr>
+            <q-grid:td key="firstName">
+                <b>{{$value}}</b>
+            </q-grid:td>
+            <q-grid:td key="dateBirth">
```

Comment:
Yes, by perf reasons we'd better use something event-driven like 
`<q-grid:td on='edit'>`

It would be great, if we could check when=expr value manually without adding watchers. But I don't think, it's possible